bin/detail/toolchain_table.py

```diff
@@ -178,6 +181,9 @@ def verify(self):
       Toolchain('ios-nocodesign-9-2', 'Xcode', ios_version='9.2', nocodesign=True),
       Toolchain('ios-nocodesign-9-2-arm64', 'Xcode', ios_version='9.2', nocodesign=True),
       Toolchain('ios-nocodesign-9-2-armv7', 'Xcode', ios_version='9.2', nocodesign=True),
+      Toolchain('ios-nocodesign-9-3', 'Xcode', ios_version='9.3', nocodesign=True),
```

Comment:
CMake part is missing. I've added it: https://github.com/ruslo/polly/commit/ce33989362e04919d9d8fb49517c76705779cf52